docs/filters.md

```diff
@@ -148,7 +148,7 @@ Matches all issues with a body (or description) containing the given string.
 
 *Expects a string*
 
-Matches all issues associated with any milestones whose names contain the given string.
+Matches all issues associated with any milestones whose names contain the given string. `current` or `curr` can be used to refer to an open milestone with earliest due date, or an overdue open milestone with open issues if any. `prev`, `prev-1`, `prev-2`, `next`, `next+1, `next+2`, etc. can be used to refer to milestones before or after `current`, sorted by due date.
 
```

Comment:
Sounds good to me.
On Oct 13, 2015 7:40 AM, "Damith C. Rajapakse" <notifications@github.com>
wrote:

> In docs/filters.md
> <https://github.com/HubTurbo/HubTurbo/pull/1050#discussion_r41813181>:
>
> > @@ -148,7 +148,7 @@ Matches all issues with a body (or description) containing the given string.
> >
> >  *Expects a string*
> >
> > -Matches all issues associated with any milestones whose names contain the given string.
> > +Matches all issues associated with any milestones whose names contain the given string. `current` or `curr` can be used to refer to an open milestone with earliest due date, or an overdue open milestone with open issues if any. `prev`, `prev-1`, `prev-2`, `next`, `next+1, `next+2`, etc. can be used to refer to milestones before or after `current`, sorted by due date.
> >
>
> Yes, it is fine to have a alias.
> Actually, why not just use current only (i.e. not use previous/next)?
> That will make it simpler.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/HubTurbo/HubTurbo/pull/1050/files#r41813181>.
>